dev/com.ibm.ws.org.apache.cxf.cxf.rt.rs.mp.client.3.3/bnd.bnd

```diff
@@ -29,7 +29,7 @@
   com.ibm.websphere.javaee.interceptor.1.2;version=latest,\
   com.ibm.websphere.org.eclipse.microprofile.config.1.3;version=latest,\
   com.ibm.websphere.org.eclipse.microprofile.faulttolerance.1.0;version=latest,\
-  com.ibm.websphere.org.eclipse.microprofile.rest.client.1.3;version=latest,\
+  io.openliberty.org.eclipse.microprofile.rest.client.2.0;version=latest,\
```

Comment:
This bundle is included in mpRestClient-1.2, 1.3 and 1.4 ... How can it have a dependency on the 2.0 API?